public/index.ejs

```diff
@@ -267,10 +291,22 @@ function updateFollowingUI(users) {
       e.preventDefault();
       history.pushState({path: newURL.toString()}, '', newURL.toString());
       updateUIToMatchURL();
+      fetchAbortHandler();
       update();
     }
   }
 }
+
+function fetchAbortHandler() {
```

Comment:
This should probably be renamed to something more specific? Maybe "abortOutstandingFetches()"?

Reply:
Sounds good.